.github/workflows/chain-types.yml

```diff
@@ -0,0 +1,28 @@
+name: Chain types
+on:
+  schedule:
+    - cron:  '30 0/12 * * *'
+
+jobs:
+  types:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 'lts/*'
+    - name: check
+      env:
+        CI_LOG: 123
+      run: |
+        yarn install --immutable
+        yarn ci:chainTypes
+
+    - name: issue
+      if: ${{ failure() }}
+      uses: JasonEtco/create-an-issue@e27dddc79c92bc6e4562f268fffa5ed752639abd
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_PAT_BOT }}
+      with:
+        filename: .github/chain-types.md
+        update_existing: true
```

Comment:
Do we absolutely want to import this in?

Reply:
Not really. Will clean this to only preserve the workflows we need

Reply:
@ETeissonniere Done. I've removed the new workflows, as we don't require it